Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSSystemInfo.m

```diff
@@ -63,6 +63,48 @@ static inline bool is_jailbroken() {
     return is_jb;
 }
 
+/**
+ * Returns the content of /System/Library/CoreServices/SystemVersion.plist
+ * bypassing the open syscall shim that would normally redirect access to this
+ * file for iOS apps running on macOS.
+ *
+ * https://opensource.apple.com/source/xnu/xnu-7195.81.3/libsyscall/wrappers/system-version-compat.c.auto.html
+ */
+static NSDictionary * bsg_systemversion() {
+    static NSDictionary *systemVersion;
+    if (!systemVersion) {
```

Comment:
Should this block be wrapped in a `dispatch_once` block? presumably it only needs to be set once.